.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- Good job 👍 adding the node_modules directory but also please make sure to add a "test.md" here as per the [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=A%20test.md,your%20GitHub%20repository.). I've also noticed you don't have the test.md file in your repo so please make sure to add it to your repo as well.